context-js/src/lib.rs

```diff
@@ -36,6 +39,11 @@ pub fn env_parse(env_vars: js_sys::Object) -> Result<Option<env::parser::CIInfo>
     Ok(ci_info_class)
 }
 
+#[wasm_bindgen]
+pub fn parse_branch_class(value: &str) -> Result<BranchClass, JsError> {
+    BranchClass::try_from(value).map_err(|e| JsError::new(&e.to_string()))
```

Comment:
optional: if you want, you can expand this here (rather than on the TS side) to also take in a pr_number and return accordingly. I would still keep the result as `BranchClass` though, as you have it

Reply:
So I attempted to add an optional number param but failed miserably LOL. If you wanna help me to get it to compile to add the optional pr number param: [`b88d957` (#159)](https://github.com/trunk-io/analytics-cli/pull/159/commits/b88d957dcd574129ddb45c9e43b14472b854048b)

Otherwise gonna skip this.

Reply:
Today was my first time ever even seeing/coding in Rust so ahem excuse my ignorance.